x-pack/plugins/upgrade_assistant/public/application/components/overview/overview.tsx

```diff
@@ -94,18 +97,24 @@ export const Overview = withRouter(({ history }: RouteComponentProps) => {
         >
           <EuiText>
             <FormattedMessage
-              id="xpack.upgradeAssistant.overview.checkUpcomingVersion"
-              defaultMessage="If you are not on the latest version of the Elastic Stack, use the Upgrade Assistant to prepare for the next upgrade."
+              id="xpack.upgradeAssistant.overview.upgradeToLatestMinorBeforeMajorMessage"
+              defaultMessage="Check the {link}. To safely proceed to the next major ensure you are on the latest minor of version."
```

Comment:
```suggestion
              defaultMessage="Check the {link}. Before upgrading to a new major version, you must first upgrade to the latest minor of the preceding major version."
```
I prefer 'upgrade' over 'proceed' as it's more explicit wording.
Another option could be to be super explicit even if that means maintaining this wording for each major release
"Before upgrading to version 9.0, you must first upgrade to version 8.18."

Reply:
Thanks that sounds good! I'm happy to go with your suggestion but I think "of the preceding major version" should be "of this major version". Seems OK given we refer to the current version on the line above, WDYT? I'll update the description with a screenshot.

> super explicit

The problem is we may decide we need an 8.19 and we've shipped a bunch of Kibana versions that only mention 8.18. But I agree, this would be the very best.